.github/workflows/test.yml

```diff
@@ -86,12 +86,8 @@ jobs:
             moodle-branch: 'MOODLE_400_STABLE'
           - php: '8.0'
             moodle-branch: 'MOODLE_311_STABLE'
-          - php: '7.4'
-            moodle-branch: 'MOODLE_310_STABLE'
           - php: '7.4'
             moodle-branch: 'MOODLE_39_STABLE'
-          - php: '7.4'
-            moodle-branch: 'MOODLE_38_STABLE'
```

Comment:
Regarding removing 38 and 310 completely... I must confess that I'm not 100% convinced... maybe we could add some variable, say, `moodle-app`(o `skip-moodle-app`) to the supported branches in the matrix and then set the  `MOODLE_APP` based on that?

That way we could keep the feature enabled and disabled at our preference, not only to disable it in the 38 and 310 branches (obviously), but to:
1. Continue having 38 and 310 covered for everything else.
2. Maybe split some run (main branch) and have it both with the app enabled and disabled (just to verify that everything continues working with the app disabled).
3. Slightly better real coverage (I'll comment later about better phpunit coverage).

Reply:
I will try to give to this a try myself on top of your branch. While it's not super-critical... I feel that it's like abandoning those 2 branches completely and prefer to keep them covered, no matter that doesn't include the new app functionality.

So, ignore this, I'll take a look. Resolved.